PlayerToken.java

```diff
@@ -0,0 +1,50 @@
+package game.logic;
+enum PlayerToken { NOPLAYER, PLAYER1, PLAYER2 };
+
+
+private class Player {
+	
+	private PlayerToken playerTokens;
+	
+	
+	private int numPieces;
+	
+	private int numPiecesOnBoard;
+	
+	private int numPiecesLeft;
+	
+	
+	
+	
+	public PlayerToken getPlayerToken() {
+		return playerTokens;
+	}
+	
+	
+	public int getPiecesTotal() {
+		return numPiecesTotal;
+	}
+	
+	
+	public int getPiecesOnBoard() {
+		return numPiecesOnBoard;
+	}
+	
+	
+	public int getPiecesLeft() {
+		return numPiecesLeft;
+	}
+	
+	
+	public void placePiece() {
+		if (numPiecesLeft > 0) {
+			numPieceOnBoard++;
+			numPiecesLeft--;
+		}
+		else {
+			System.out.print("You are out of pieces to place")); 
```

Comment:
If it is considered Illegal to place a piece when there are no pieces left to place, we should not be printing to the system output. Instead we should be raising an error. Please update this to raise an error

Reply:
This could also be solved quickly by change the function to a Boolean, but we would have to change it to an error output next sprint.